pkg/runtime/executor.go

```diff
@@ -51,8 +52,24 @@ func NewExecutor(iacType, iacVersion string, cloudType []string, filePath, dirPa
 		iacType:    iacType,
 		iacVersion: iacVersion,
 		configFile: configFile,
-		scanRules:  scanRules,
-		skipRules:  skipRules,
+	}
+
+	// read config file and update scan and skip rules
+	if err := e.initRuleSetFromConfigFile(); err != nil {
```

Comment:
we should add tests for the new behaviour that we added i.e. command line values overrides config file values.